go/mysql/slave_status.go

```diff
@@ -40,8 +42,13 @@ func (s *SlaveStatus) SlaveRunning() bool {
 
 // SlaveStatusToProto translates a Status to proto3.
 func SlaveStatusToProto(s SlaveStatus) *replicationdatapb.Status {
+	var relayLogPosition string
+	if s.RelayLogPosition != nil {
+		relayLogPosition = EncodePosition(*s.RelayLogPosition)
+	}
 	return &replicationdatapb.Status{
 		Position:            EncodePosition(s.Position),
+		RelayLogPosition:    relayLogPosition,
```

Comment:
`EncodePosition` already handles encoding the zero value (equivalent of nil pointer).

```suggestion
	return &replicationdatapb.Status{
		Position:            EncodePosition(s.Position),
		RelayLogPosition:    EncodePosition(s.RelayLogPosition),
```

Reply:
Done.